README.md

```diff
@@ -341,16 +341,20 @@ By default, MusicBrainz Server uses 10 `plackup` processes at once.
 This number can be changed using the Docker environment variable
 `MUSICBRAINZ_SERVER_PROCESSES`.
 
-#### Customize FTP mirror
+#### Customize HTTP/FTP mirror
 
 By default, data dumps and pre-built search indexes are downloaded from
-`ftp://ftp.eu.metabrainz.org/pub/musicbrainz`.
+`http://ftp.eu.metabrainz.org/pub/musicbrainz`.
 
-The FTP mirror can be changed using the Docker environment variable
-`MUSICBRAINZ_BASE_FTP_URL`.
+The HTTP mirror can be changed using the Docker environment variable
+`MUSICBRAINZ_BASE_HTTP_URL`.
 
-See also the [list of mirrors](https://musicbrainz.org/doc/MusicBrainz_Database/Download#Download);
-Only FTP URLs are supported here.
+An FTP mirror can be used instead by specifying the `MUSICBRAINZ_BASE_FTP_URL`
+Docker environment variable. Note that support for FTP mirrors is deprecated
+and will be removed in a future release.
```

Comment:
The option `--base-ftp-url` might be deprecated, but not the support for FTP as there are FTP mirrors.

For example we could replace both options with `--base-url` later on and support FTP/HTTP/RSYNC.